pkg/config/derive.go

```diff
@@ -223,3 +226,55 @@ func (derived) MaxDataDisksPerVM(cs *api.OpenShiftManagedCluster) (string, error
 
 	return "", fmt.Errorf("unknown VMSize %q", app.VMSize)
 }
+
+func (derived) MDSDConfig(cs *api.OpenShiftManagedCluster) (string, error) {
+	var tmpl = `<?xml version="1.0" encoding="utf-8"?>
+    <MonitoringManagement version="1.0" namespace="{{ .Namespace | Escape }}" eventVersion="1" timestamp="2017-08-01T00:00:00.000Z">
+        <Accounts>
+            <Account moniker="{{ .Account | Escape }}" isDefault="true" autoKey="false" />
+        </Accounts>
+        <Management eventVolume="Large" defaultRetentionInDays="90" >
```

Comment:
TODO: Check if pass region (cluster region, not RP region) and make sure we log and metric same

Reply:
putting a note for passing in OS_ rg also

Reply:
is `OS_rg` is not the same as `ResourceGroupName` ?

Reply:
`ResourceGroupName` is the one with uuid i believe, `OS_...` rg is the one that contains the managed app resource

Reply:
Just thinking, how this name is accessible to the plugin?

Reply:
looks like i missed this comment. not sure if plugin can access from somewhere, rp can pass this.